modules/programs/borgmatic.nix

```
@@ -56,10 +81,21 @@ let
         };
 
         repositories = mkOption {
-          type = types.listOf types.str;
-          description = "Paths to repositories.";
-          example =
-            literalExpression ''["ssh://myuser@myrepo.myserver.com/./repo"]'';
+          type = types.listOf (types.either types.str repositoryOption);
```

@rycee suggested to use `apply` here instead of deferring cleaning until much later in the file:

```suggestion
          type = types.listOf (types.either types.str repositoryOption);
          apply = cleanRepositories;
```